packages/docusaurus/package.json

```
@@ -44,7 +44,7 @@
     "@docusaurus/cssnano-preset": "2.0.0-beta.15",
     "@docusaurus/logger": "2.0.0-beta.15",
     "@docusaurus/mdx-loader": "2.0.0-beta.15",
-    "@docusaurus/react-loadable": "5.5.2",
+    "@docusaurus/react-loadable": "npm:@react-loadable/revised@0.0.26",
```

```suggestion
    "@react-loadable/revised": "0.0.26",
```

Also why not using the latest version? 🤔

---

Curious why we had this package alias? To trick TS and get the correct typedefs?

---

which package alias? `@docusaurus/react-loadable` is our fork, we can't use just `react-loadable`

---

Yes I know, but we still imported `react-loadable` in the code. Is that so we get the correct type defs from `@types/react-loadable`?